src/client/content_mapblock.cpp

```diff
@@ -381,12 +381,12 @@ void MapblockMeshGenerator::drawAutoLightedCuboid(aabb3f box, const f32 *txc,
 		box.MinEdge *= f->visual_scale;
 		box.MaxEdge *= f->visual_scale;
 	}
-	box.MinEdge += origin;
-	box.MaxEdge += origin;
```

Comment:
If this introduces a new requirement that origin must be added *after* calling `drawNodeboxNode` that should be written down where `drawNodeboxNode` is defined or so.

Reply:
Origin is added in line 388 below, I just moved generation of texture coords earlier, because having origin added breaks it.

Reply:
It does not. I merely moved generation of texture coordinates earlier because it was broken by adding origin too early.